tasks/14.txt

```diff
@@ -1,3 +1,5 @@
 Create a SQL query to get Product list of stock is less than the quantity on order:
 
-| ProductName | UnitsInOrder| UnitsInStock |
\ No newline at end of file
+| ProductName | UnitsInOrder| UnitsInStock |
+
+db.getCollection('products').find({ $where:function(){ return this.UnitsInStock < this.UnitsOnOrder} })
```

Comment:
Please do not use $where. It is very slow. Only for some places with really small collections or data sets